src/content/repositories/upstreams/index.mdx

```diff
@@ -5,6 +5,7 @@ import upstream_formats from './images/upstream_formats.png'
 import edit_upstream from './images/edit_upstream.png'
 
 import cargo from './images/cargo.png'
+import conda from './images/conda.png'
```

Comment:
The screenshot for the Conda upstream shows an incorrect URL. For Anaconda, it's `https://repo.anaconda.com/pkgs/main`. It's important to specify the channel when specifying the upstream. 

For other upstreams like Condaforge/Bioconda, the URL would look like: `https://conda.anaconda.org/conda-forge/` or `https://conda.anaconda.org/bioconda/`